### PR TITLE
Feature/fix issue

### PR DESCRIPTION
**Pull request by yoution**
@maxceem three issues  are merged in this one commit

### CODE REVIEW

**Review by maxceem — CHANGES_REQUESTED** [review]
@yoution all works good.

There are a few code improvements, see comments below.

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "dev": "webpack-dev-server --port 8501 --host 0.0.0.0",
     "dev-https": "webpack-dev-server --https --port 8501 --host 0.0.0.0",
     "build": "webpack --mode=${APPMODE:-development} --env.config=${APPENV:-dev}",
+    "build:prod": "webpack --mode=${APPMODE:-production} --env.config=${APPENV:-prod}",
```
Comment (maxceem) [inline package.json:8]: @yoution we don't need to add the second `build:prod` as we can just set env variable `APPMODE=production` and run `npm run build`, see https://github.com/topcoder-platform/taas-app/issues/13#issuecomment-754891697

**Comment by yoution** [comment]
@maxceem please review

**Review by maxceem — APPROVED** [review]
Thanks for the quick update @yoution. Perfect now.

**Pull request merged**